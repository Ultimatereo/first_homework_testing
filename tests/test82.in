5
-860519241763 111007311863 -405705848587 671935130910 937213642966 900417807759
683819495696 104202949406 63760271734 -960191914980 939594047504 631999343483
78851047525 -375582962426 84436080887 -220884949895 328880264521 620096955616
-486828447709 233352930290 -157928922424 -878445530126 446614767479 528862301967
-191914393899 903308959863 221611160990 358768799378 638495485690 -285954851117
