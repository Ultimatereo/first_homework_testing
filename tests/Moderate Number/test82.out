-0.364193
-0.892211
-0.0737892
-0.196423
0.840909
